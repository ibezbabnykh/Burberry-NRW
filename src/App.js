import React from 'react';
import { BrowserRouter as Router } from 'react-router-dom';

import './App.scss';

import { Header } from 'components/common';
import AppRoutes from 'AppRoutes';

const App = () => {
  return (
    <div className="App">
      <Router>
        <Header />
        <main className="main center-holder">
          <AppRoutes />
        </main>
      </Router>
    </div>
  );
}

export default App;