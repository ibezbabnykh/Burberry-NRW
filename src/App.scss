@import 'assets/style/variables.scss';

.main {
  padding-top: 30px;
  padding-bottom: 30px;
}

.center-holder {
  max-width: 1024px;
  padding-left: 20px;
  padding-right: 20px;
  margin: 0 auto;
}