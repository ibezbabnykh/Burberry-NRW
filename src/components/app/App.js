import React from 'react';

import './App.css';

const App = () => {
  return (
    <div className="App">
      Burberry NRW
    </div>
  );
}

export default App;