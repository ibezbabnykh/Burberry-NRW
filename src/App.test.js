import App from './App';

describe('App component', () => {
  it('should render', () => {
    const component = toJson(shallow(<App />));
    expect(component).toMatchSnapshot();
  });
});