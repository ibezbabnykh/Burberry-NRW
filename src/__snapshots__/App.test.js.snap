// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component should render 1`] = `
<div
  className="App"
>
  <BrowserRouter>
    <Header />
    <main
      className="main center-holder"
    >
      <AppRoutes />
    </main>
  </BrowserRouter>
</div>
`;
